src/preamble.tex

```diff
@@ -16,14 +16,16 @@
 % Uncomment for diagnostics!
 % \usepackage[cam,width=6in,height=9in]{crop}
 
+\usepackage{comment}
+
 % Font settings
 %--------------------------------------------------------------%  Note:
 \usepackage[no-math]{fontspec}                                 %  This exact order
 \usepackage{libertine}                                         %  of declarations
 \defaultfontfeatures{%                                         %  is important for
   Scale=MatchLowercase, % needed here ...                      %  single quotes not
 }                                                              %  to turn out curly
-\setmonofont[BoldFont=Inconsolata-LGC-Bold.ttf]{Inconsolata-LGC-Bold.ttf} %  ("typographic")
+\setmonofont[BoldFont=JuliaMono-Medium.ttf]{JuliaMono-Medium.ttf} %  ("typographic")
```

Comment:
JuliaMono-Medium seems to support more unicode symbols, in particular, the "ring" symbol we use to denote composition (`\circ`) now appears as it should in the Agda code snippets.

Reply:
Nice! I will check on my machine.